etc/jitsi-meet-desktop.profile

```diff
@@ -0,0 +1,36 @@
+# Firejail profile for jitsi-meet-desktop
+# Description: Jitsi Meet desktop application powered by Electron
+# This file is overwritten after every install/update
+# Persistent local customizations
+include jitsi-meet-desktop.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec /tmp
+
+noblacklist ${HOME}/.config/Jitsi Meet
+
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/Jitsi Meet
+whitelist ${HOME}/.config/Jitsi Meet
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-runuser-common.inc
+include whitelist-var-common.inc
+
+nowhitelist ${DOWNLOADS}
```

Comment:
I couldn't find any ```nowhitelist``` command in ```profile.template```. Is this the right place to do this, or should I put it with the ```whitelist``` commands?

Reply:
I recently added it just before the whitelist-block.

Reply:
Moved. I've also put in some newlines to separate sections.